components/badge/w-badge.vue

```diff
@@ -0,0 +1,39 @@
+<script setup>
+defineOptions({
+  name: 'wBadge',
+});
+
+import { badge as ccBadge } from '@warp-ds/css/component-classes';
+import { computed } from 'vue';
+
+const props = defineProps({
+  as: { type: String, default: 'div' },
+  type: {
+    type: String,
+    default: 'neutral',
+    validator: (value) => ['neutral', 'info', 'positive', 'warning', 'negative', 'disabled', 'notification', 'price'].includes(value)
+  },
+  position: {
+    type: String,
+    validator: (value) => ['tl', 'tr', 'br', 'bl'].includes(value)
+  },
+});
+
+const badgeClasses = computed(() => [
+  ccBadge.base,
+  ccBadge[props.type],
+  {
+    [ccBadge.positionBase]: props.position,
+    [ccBadge.positionTL]: props.position === 'tl',
+    [ccBadge.positionTR]: props.position === 'tr',
+    [ccBadge.positionBR]: props.position === 'br',
+    [ccBadge.positionBL]: props.position === 'bl',
```

Comment:
Now that we know that Badge was previously a Tag, and that component had `topLeft`, `topRight`, `bottomLeft` and `bottomRight` in its API, would it make sense to use those? It always takes me a second to understand which position tl, tr, br and bl stand for, but that's just me 😅

Reply:
In the tag component they are separate properties, which means that you can add both `bottomLeft` and `topRight`. That is confusing to me. But sure, changing to position="top-left|top-right|bottom-right|bottom-left" would be easier to understand. 👍

Reply:
> In the tag component they are separate properties, which means that you can add both bottomLeft and topRight.*
I don't think those were ever intended to be used together. I see there are other examples of such boolean attributes in Vue components that were not really supposed to be used together either, but it was technically that was possible 😊